backend/translation/serializers.py

```diff
@@ -5,14 +5,10 @@
 
 class TranslationSerializer(serializers.ModelSerializer):
     translation_type = serializers.SerializerMethodField()
-    target_language = serializers.SerializerMethodField()
```

Comment:
Why is this removed? Without the serializer having the language, how will the language be passed via the response?

Reply:
Discussed on call.